packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx

```diff
@@ -71,12 +73,18 @@ export const AccountsMenu = () => {
             <Header>
                 <ExpandedSidebarOnly>
                     <Row justifyContent="space-between" gap={spacings.xs}>
-                        {!isEmpty && <AccountSearchBox />}
-                        <AddAccountButton
-                            isFullWidth={isEmpty}
-                            data-testid="@account-menu/add-account"
-                            device={device}
-                        />
+                        {isDiscoveryRunning ? (
+                            <SkeletonRectangle animate width="100%" height={38} />
+                        ) : (
+                            <>
+                                {!isEmpty && <AccountSearchBox />}
+                                <AddAccountButton
+                                    isFullWidth={isEmpty}
```

Comment:
I believe you don't need this prop anymore. It shouldn't be full width anymore

Reply:
Not my change!! :D

Reply:
Invalid: there can be no accounts